README.md

```diff
@@ -69,3 +69,537 @@ A nearly-vanilla sufia7 app with the potential to be built out as GW ScholarSpac
+	```
+*  Create a symbolic link to `libtcnative1.so` to address a Ubuntu/Tomcat bug
+        
+        % sudo ln -sv /usr/lib/x86_64-linux-gnu/libtcnative-1.so /usr/lib/
+
+*  Replace the `web.xml` files for Fedora with the `web_ssl.xml` files from the repo:
+
+        % cd /opt/install
+        Retrieve `web_ssl.xml` from `tomcat_conf/fcrepo-webapp/web_ssl.xml` in the GitHub repo
+	      % cp tomcat_conf/fcrepo-webapp/web_ssl.xml /var/lib/tomcat7/webapps/fcrepo/WEB-INF/web.xml
+
+
+*  Restart Tomcat and test access over HTTPS
+
+        % sudo service tomcat7 restart
+
+### Application server setup:
+
+### Dependencies
+
+* Install ubuntu 14.04 LTS package dependencies:
+
+        % sudo apt-get update
+        % sudo apt-get install git postgresql libpq-dev redis-server unzip clamav-daemon curl imagemagick libapache2-mod-shib2  libreoffice libcurl4-openssl-dev apache2-threaded-dev apache2-mpm-worker apache2-threaded-dev 
+
+* Install RVM for multi-users
+
+        % curl -L https://get.rvm.io | sudo bash -s stable
+        % source ~/.rvm/scripts/rvm
+        % rvm install ruby-2.3.0
+        % sudo nano /etc/group
+        
+        Add users to the rvm group
+
+* Install Rails:
+
+        % gem install rails -v 4.2.6 -N
+        
+* Create the directory structure
+
+        % mkdir /opt/scholarspace
+        % mkdir /opt/install
+        % mkdir /opt/xsendfile
+        
+* Create the necessary user groups and assign folder permissions
+
+        % sudo groupadd scholarspace
+        
+        Edit `/etc/group` and add users (including www-data) to the new group
+        
+        % sudo chown $USER:scholarspace /opt/scholarspace
+        % sudo chown www-data:www-data /opt/xsendfile
+
+* Get the GW ScholarSpace code:
+
+        % cd /opt/scholarspace
+        % git clone https://github.com/gwu-libraries/scholarspace-sufia7.git
+
+* Install gems
+
+        % cd scholarspace-sufia7
+        % bundle install --without development --deployment
+
+* Create a postgresql user
+
+        % sudo su - postgres
+        (postgres)% psql
+        postgres=# create user YOURDBUSERNAME with createdb password 'YOURSFMDBPASSWORD';
+
+  The result should be:
+
+        CREATE ROLE
+
+* Create three databases (e.g. scholarspace_dev, scholarspace_test, scholarspace_prod)
+
+        postgres=# \q
+        (postgres)% createdb -O YOURDBUSERNAME YOURDEVDBNAME
+        (postgres)% createdb -O YOURDBUSERNAME YOURTESTDBNAME
+        (postgres)% createdb -O YOURDBUSERNAME YOURPRODDBNAME
+        (postgres)% exit
+
+* Create the `database.yml` file
+
+        % cd config
+        % cp database.yml.template database.yml
+
+  Edit `database.yml` to add your specific database names and credentials
+
+* Create the `solr.yml` file
+
+        % cd config
+        % cp solr.yml.template solr.yml
+
+  Edit `solr.yml` to add your specific names and credentials
+
+* Create the `blacklight.yml` file
+
+        % cd config
+        % cp blacklight.yml.template blacklight.yml
```

Comment:
There is a blacklight.yml file, not a blacklight.yml.template file.

What are correct values for dev, test, prod?